### PR TITLE
Fix the name of real_time_correlative metric in 2D.

### PR DESCRIPTION
**Pull request by wohe**
This was called fast_correlative before, but the metric was about
the score of the real-time correlative scan matcher in local SLAM.

### CODE REVIEW

File: cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc [hunk 1]
```diff
@@ -82,7 +82,7 @@ std::unique_ptr<transform::Rigid2d> LocalTrajectoryBuilder2D::ScanMatch(
         pose_prediction, filtered_gravity_aligned_point_cloud,
```
Comment (schwoere) [inline cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc]: please in line 81. Add const:
`const double score =`

Comment (wohe) [reply cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc]: Done.

**Comment by wohe** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user wohe. Merge queue now has a length of 2.

**Pull request merged**